js/src/admin/components/PermissionGrid.js

```diff
@@ -210,6 +210,12 @@ export default class PermissionGrid extends Component {
       permission: 'discussion.delete'
     }, 80);
 
+    items.add('postWithoutThrottle', {
+      icon: 'fas fa-swimmer',
```

Comment:
What's the idea behind [that icon](https://fontawesome.com/icons/swimmer?style=solid)? Swimming with the flood? :laughing:

Reply:
Floodgate? No idea, it's what I originally chose 🤔